### PR TITLE
Added translation keys for PayPalExpress payment method

### LINKED ISSUE #5302: Missing translation keys in Paypal Express form
## Description

When adding paypal as a payment method, all the fields a user needs to fill are not translated.

The **title "provider settings"** is also not transalted.

## Steps to Reproduce
1. As a enterprise manager, go to `/admin/payment_methods/ENTERPRISEID/edit`
2. Select Paypal Express as a provider
3. See the missing translation

## Animated Gif/Screenshot
image

## Severity

bug-s3: a feature is broken but there is a workaround

## Your Environment

* Version used: v2.9.5
* Browser name and version: Firefox 75
* Operating System and version (desktop or mobile): Desktop

### PR DESCRIPTION
**Pull request by rmklaus12**
#### What? Why?

Closes #5302 

When viewed in French, the field headings for the PayPalExpress payment method were not translated. The 'Provider Settings' heading was also not translated.

#### What should we test?
The New Payment Method at `/admin/payment_methods/new` in French to ensure all fields and headings are properly translated. 

#### Release notes
Added additional translation to the New Payment method page

Changelog Category: Added

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -2952,6 +2957,14 @@ See the %{link} to find out more about %{sitename}'s features and to start using
...
     capture: "Capture"
     void: "Void"
+    login: "Login"
+    password: "Password"
+    signature: "Signature"
+    solution: "Solution"
+    landing_page: "Landing Page"
+    server: "Server"
+    test_mode: "Test Mode"
+    logourl: "Logourl"
```
Comment (luisramos0) [inline config/locales/en.yml:2967]: there's already a group for payment methods here, can you move this to below line 3042? Thanks!

File: config/locales/en.yml [hunk 2]
```diff
@@ -2380,6 +2380,11 @@ See the %{link} to find out more about %{sitename}'s features and to start using
   shipping: "Shipping"
   shipping_methods: "Shipping Methods"
   payment_methods: "Payment Methods"
+  provider_settings: "Provider Settings"
+  active: "Active"
+  description: "Description"
+  tags: "Tags"
+  provider: "Provider"
```
Comment (luisramos0) [inline config/locales/en.yml:2387]: Sorry about this but I have handled this problem in a very recent PR:
https://github.com/openfoodfoundation/openfoodnetwork/pull/5425/files
I have actually moved these translations around so it's better if you delete these 5 keys from this PR. They will become obsolete after 5425.

Btw, if you want to learn something for next time, as we fix translations, we also try to make translations use lazy lookups, see here for details:
https://github.com/openfoodfoundation/openfoodnetwork/wiki/Internationalisation-(i18n)#development
One example is here:
https://github.com/openfoodfoundation/openfoodnetwork/pull/5425/files#diff-1985e8d0aa61624ade9de3f91944c7b3R19

File: config/locales/en.yml (hunk 1 above)
Comment (luisramos0) [reply config/locales/en.yml:2967]: I wonder where these translations are called from? If you know you could try to make them lazylookups and put them in their relative path. It's a stretch goal, maybe you want to do only in the next PR :+1:

File: config/locales/en.yml (hunk 2 above)
Comment (rmklaus12) [reply config/locales/en.yml:2387]: Thank you for the feedback @luisramos0. 

> Sorry about this but I have handled this problem in a very recent PR:
> https://github.com/openfoodfoundation/openfoodnetwork/pull/5425/files
> I have actually moved these translations around so it's better if you delete these 5 keys from this PR. They will become obsolete after 5425.
> 
> Btw, if you want to learn something for next time, as we fix translations, we also try to make translations use lazy lookups, see here for details:
> https://github.com/openfoodfoundation/openfoodnetwork/wiki/Internationalisation-(i18n)#development
> One example is here:
> https://github.com/openfoodfoundation/openfoodnetwork/pull/5425/files#diff-1985e8d0aa61624ade9de3f91944c7b3R19

I had been curious about how/when the lazy lookup is implemented so yes, always want to learn for next time :-)

Comment (rmklaus12) [reply config/locales/en.yml:2387]: @luisramos0  - your comment mentioned removing 5 keys, but the code snippet shows 4. Can you please clarify which ones you want me to remove? Thanks!

Comment (luisramos0) [reply config/locales/en.yml:2387]: the snippet is 4 lines long, you have changed 5 lines, see in https://github.com/openfoodfoundation/openfoodnetwork/pull/5448/files

Comment (rmklaus12) [reply config/locales/en.yml:2387]: @luisramos0. Deleted and will push again. Thanks

**Comment by filipefurtad0** [comment]
Hey @rmklaus12,

image

The fields look fine, translations should take effect.
Thank you!

**Pull request merged**